src/test/App.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import App from '../App';
+
+it('App renders correctly', () => {
+  // eslint-disable-next-line no-undef
+  const tree = renderer
+    .create(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/172781175-6828ef11-5ef6-4298-8e17-bd2cd842959d.png)

- [ ] This test is failing also because you forgot to import the renderer from `react-test-rendere`

```test.js
import renderer from 'react-test-renderer';
```

Reply:
Thank you.